monai/networks/layers/convolutions.py

```diff
@@ -1,73 +1,95 @@
-import torch.nn as nn
 import numpy as np
+import torch.nn as nn
 
-from monai.utils.convutils import samePadding
-from monai.networks.utils import getConvType, getDropoutType, getNormalizeType
+from monai.networks.utils import (get_conv_type, get_dropout_type, get_normalize_type)
+from monai.utils.convutils import same_padding
 
 
 class Convolution(nn.Sequential):
-    def __init__(self, dimensions, inChannels, outChannels, strides=1, kernelSize=3, instanceNorm=True, 
-                 dropout=0, dilation=1, bias=True, convOnly=False, isTransposed=False):
+
+    def __init__(self,
```

Comment:
This is just really ugly. There's no need to put every argument on a separate line when the line is long, just break it as the original was. If this is a pycharm thing it surely can be turned off

Reply:
This layout makes it very simple to see new changes in the function definition in Git, and it's very readable. I'd even put the final bracket in a new line.

Reply:
It's even more important when typing is used. See BDFL's comment here: https://github.com/python/typing/issues/433#issuecomment-302452660

Reply:
I see it just wasting a huge amount of space, it impacts readability I find having to scroll around more to see what code does what.

Reply:
agree with @fepegar. we need another round of voting... since with automated code formatting tool we can easily go from one to the other, it’s not a high priority issue for now.